### DISCUSSION TITLE
substrate.get_metadata_modules How to output the specific call_functions name under count_call_functions?

### DISCUSSION THREAD

**Discussion by yytomives** [polkascan/py-substrate-interface#289] [Q&A] [answered]
substrate.get_metadata_modules currently only prints count_call_functions, how can I print out the call_functions name of the corresponding module?
for example,
{'metadata_index': 0, 'module_id': 'System', 'name': 'System', 'spec_version': 2022072701, 'count_call_functions': 9, 'count_storage_functions': 16, 'count_events': 6, 'count_constants': 6, 'count_errors': 6} ,How to output the specific call_functions name under count_call_functions?

**Comment by arjanz** [comment, marked as answer]
With `substrate.get_metadata_call_functions()` and `substrate.get_metadata_call_function(pallet_name, function_name)`, for more info: https://polkascan.github.io/py-substrate-interface/#substrateinterface.SubstrateInterface.get_metadata_call_functions
